bundles/org.openhab.binding.tidal/README.md

```diff
@@ -0,0 +1,293 @@
+| trackName       | String    | Read-only  | The name of the currently playing track.                                                          |
+| trackDuration   | String    | Read-only  | The duration (m:ss) of the currently playing track. This is updated every second.                 |
+| trackDurationMs | Number    | Read-only  | The duration of the currently playing track in milliseconds.                                      |
+| trackProgress   | String    | Read-only  | The progress (m:ss) of the currently playing track. This is updated every second.                 |
+| trackProgressMs | Number    | Read-only  | The progress of the currently playing track in milliseconds.                                      |
+| playlists       | Selection | Read-write | This channel will be populated with the users playlists. Set the playlist ID to start.            |
+| playlistName    | String    | Read-write | The currently playing playlist. Or empty if no playing list is playing.                           |
+| albumName       | String    | Read-only  | Album Name of the currently playing track.                                                        |
+| albumImage      | RawType   | Read-only  | Album Image of the currently playing track.                                                       |
+| albumImageUrl   | String    | Read-only  | Url to the album Image of the currently playing track.                                            |
+| artistName      | String    | Read-only  | Artist Name of the currently playing track.                                                       |
+
+The `playlists` channel has 2 parameters:
+
+| Parameter | Description                                                                |
+| --------- | -------------------------------------------------------------------------- |
+| offset    | The index of the first playlist to return. Default `0`, max `100.000`      |
+| limit     | The maximum number of playlists to return. Default `20`, min `1`, max `50` |
+
+The `albumImage` and `albumImageUrl` channels has 1 parameter:
+
+| Parameter  | Description                                                                                |
+| ---------- | ------------------------------------------------------------------------------------------ |
+| imageIndex | Index in list of to select size of the image to show. 0:large (default), 1:medium, 2:small |
+
+Note: The `deviceName` and `playlist` channels are Selection channels.
+They are dynamically populated by the binding with the user specific devices and playlists.
+
+**Advanced Channels:**
+
+| Channel Type ID | Item Type | Read/Write | Description                                                 |
+| --------------- | --------- | ---------- | ----------------------------------------------------------- |
+| accessToken     | String    | Read-only  | The current accessToken used in communication with Web API. |
```

Comment:
The accessToken should not be published on the bus (and send everywhere), please remove.